HLTrigger/Tools/python/convertToRaw.py

```diff
@@ -0,0 +1,125 @@
+# convert the .raw data will appear under
+# store/raw/Run2022A/MinimumBias/RAW/v1/000/run353087
```

Comment:
What are these two lines about? Should not always be the case....

Reply:
Ops, sorry, that's a leftover from the earlier developments... removed, and replaced with a minimal description.